contents/docs/data-warehouse/query.mdx

```diff
@@ -9,75 +9,42 @@ availability:
-  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/trend_dark_77178b53cb.png"
-  alt="Selecting a data warehouse table in a trend" 
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/Clean_Shot_2025_08_26_at_13_07_45_2x_751fc31300.png" 
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/Clean_Shot_2025_08_26_at_13_07_45_2x_751fc31300.png"
+  alt="SQL editor" 
   classes="rounded"
 />
 
-The main difference from events is that you must identify 3 fields from the data warehouse data to visualize it:
-
-- ID: A field that corresponds to the ID of the element. 
-- Distinct ID: A field that corresponds to an ID representing the user associated with the element. If none, can match ID.
-- Timestamp: A timestamp field representing when the element was created.
- 
-It's infeasible for us to map these fields. We pre-fill with a best guess, but allow you to edit it.
-
-### Filters and breakdowns
+Here you can:
 
-When using data warehouse tables in insights, you can use properties from those tables filter and breakdown as you would with any other insight. For example, you could filter `zendesk_tickets` for ones where the `status` is `open` or `hubspot_companies` where `lifecyclestage` is `lead`.
+1. Write your SQL query using your table like `SELECT * FROM stripe.prod.charge`
+2. Click **Run** to see the results.
+3. Modify your query using [SQL commands](/docs/data-warehouse/sql) and [functions](/docs/data-warehouse/sql/functions) as needed to get the data you want like `select amount / 100 as dollar_charge, status from stripe.prod.charge`.
 
 <ProductScreenshot
-  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/bd_light_989c851f4b.png" 
-  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/bd_dark_ef8b5baf43.png"
-  alt="Filtering and breaking down data warehouse data" 
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/Clean_Shot_2025_08_26_at_13_19_11_2x_b1d7ce399f.png" 
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/Clean_Shot_2025_08_26_at_13_18_51_2x_e62db75b67.png" 
+  alt="Querying a source in PostHog" 
   classes="rounded"
 />
 
-> **Want to create insights with a combination of PostHog and external data?** Check out our docs on [joining tables and persons](/docs/data-warehouse/join).
-
-## Querying sources with SQL
-
-For more complicated queries, you can use [SQL insights](/docs/product-analytics/sql). 
+4. Click **Create insight**.
+5. Choose and customize your visualization.
+6. Click **Save insight**.
 
-To start, create a new SQL insight from the [SQL tab](https://us.posthog.com/sql). Here you can see the schema of all available external and PostHog tables as well as saved views.
+<CalloutBox icon="IconInfo" title="Having trouble writing SQL?" type="fyi">
```

Comment:
You're missing the import so it's just rendering as text
<img width="696" height="137" alt="Screenshot 2025-08-26 at 2 56 17 PM" src="https://github.com/user-attachments/assets/dedac523-ca86-477d-991b-85fa858cc71b" />